hudi-utilities/src/test/resources/delta-streamer-config/short_trip_uber_config.properties

```diff
@@ -25,3 +25,4 @@ hoodie.datasource.hive_sync.table=short_trip_uber_hive_dummy_table
 hoodie.datasource.write.keygenerator.class=org.apache.hudi.utilities.deltastreamer.TestHoodieDeltaStreamer$TestTableLevelGenerator
 hoodie.deltastreamer.schemaprovider.registry.baseUrl=http://localhost:8081/subjects/
 hoodie.deltastreamer.schemaprovider.registry.urlSuffix=-value/versions/latest
+hoodie.deltastreamer.transformer.class=org.apache.hudi.utilities.transform.SqlFileBasedTransformer
```

Comment:
Looks there's test failure:

```java
2023-05-10T10:02:49.3996412Z [ERROR] Errors: 
2023-05-10T10:02:49.4001259Z [ERROR]   TestHoodieMultiTableDeltaStreamer.testMultiTableExecutionWithKafkaSource:187 » NullPointer
2023-05-10T10:02:49.4002065Z [ERROR]   TestHoodieMultiTableDeltaStreamer.testMultiTableExecutionWithParquetSource:231->syncAndVerify:300 » NullPointer
```
Relate to this change?

Reply:
@boneanxs Where do you see these errors? I could see all tests passed before merging.

Reply:
This is a misconfig - this transformer needs other config to work properly, without which it'll break table creation for the failing testcases. Fixing it here https://github.com/apache/hudi/pull/8705